src/CommunityToolkit.Maui.Maps/Handler/Map/MapHandler.Windows.cs

```diff
@@ -57,26 +58,45 @@ protected override FrameworkElement CreatePlatformView()
 			throw new InvalidOperationException("You need to specify a Bing Maps Key");
 		}
 
-		var mapPage = GetMapHtmlPage(MapsKey);
+		MapPage = GetMapHtmlPage(MapsKey);
 		var webView = new MauiWebView(new WebViewHandler());
-		webView.NavigationCompleted += HandleWebViewNavigationCompleted;
-		webView.WebMessageReceived += WebViewWebMessageReceived;
-		webView.LoadHtml(mapPage, null);
-
+	
 		return webView;
 	}
 
 	/// <inheritdoc />
 	[RequiresDynamicCode("Calls System.Text.Json.JsonSerializer.Deserialize<TValue>(String, JsonSerializerOptions)")]
 	[RequiresUnreferencedCode("Calls System.Text.Json.JsonSerializer.Deserialize<TValue>(String, JsonSerializerOptions)")]
 #pragma warning disable IL2046 // 'RequiresUnreferencedCodeAttribute' annotations must match across all interface implementations or overrides.
+#pragma warning disable IL3051 // 'RequiresDynamicCodeAttribute' annotations must match across all interface implementations or overrides.
+	protected override void ConnectHandler(FrameworkElement platformView)
+#pragma warning restore IL3051 // 'RequiresDynamicCodeAttribute' annotations must match across all interface implementations or overrides.
+#pragma warning restore IL2046 // 'RequiresUnreferencedCodeAttribute' annotations must match across all interface implementations or overrides.
```

Comment:
Let's use the Annotations here instead of `#pragma`, like Gerald fixed in this PR: https://github.com/CommunityToolkit/Maui/pull/2461